networking/src/p2p/peer.rs

```diff
@@ -266,9 +469,31 @@ impl Actor for Peer {
 
     fn post_stop(&mut self) {
         self.net.rx_run.store(false, Ordering::Release);
+        self.quota_update_stop.notify_one();
     }
 
     fn pre_start(&mut self, ctx: &Context<Self::Msg>) {
+        // quota replenish task
+
+        let throttle_quota = self.throttle_quota.clone();
+        let log = ctx.system.log().clone();
+        let stop = self.quota_update_stop.clone();
+        self.tokio_executor.spawn(async move {
+            loop {
+                tokio::select! {
+                    _ = tokio::time::sleep(Duration::from_millis(*THROTTLING_QUOTA_RESET_MS)) => {
+                        match throttle_quota.lock() {
+                            Ok(ref mut quota) => quota.reset_all(),
+                            Err(_) => error!(log, "Failed to obtain a lock on throttling quota"),
```

Comment:
@akoptelov 
supressed erorr, please add to log, and change everywhere error! to warn! please